src/composer.ts

```diff
@@ -520,6 +520,37 @@ export class Composer<C extends Context> implements MiddlewareObj<C> {
         return this.filter(Context.has.inlineQuery(trigger), ...middleware);
     }
 
+    /**
+     * Registers middleware for the chosen inline result of an inline query that
+     * was chosen by the user and sent to their chat partner. Telegram sends an
+     * inline query to your bot whenever a user types “@your_bot_name ...” into
+     * a text field in Telegram. You bot will then receive the entered search
+     * query and can respond with a number of results (text, images, etc) that
+     * the user can pick from to send a message _via_ your bot to the respective
+     * chat. Check out https://core.telegram.org/bots/inline to read more about
+     * inline bots.
```

Comment:
The first sentence feels a bit complicated to me. Normally I assume that the first sentence gives a broad explanation about the topic why this exists so its easier to understand the area where this function is relevant. Maybe something like "Registers middleware for InlineQueryResults by the given id or ids."

Then the text should continue to explain the Result, not the inline query in general. I assume its copy pasted and work in progress?